front-end/.eslintrc.json

```diff
@@ -49,7 +49,7 @@
         "react-hooks/rules-of-hooks": "error",
         "react-hooks/exhaustive-deps": "warn",
         "semi": [2, "always"],
-        "simple-import-sort/sort": "error",
+        "simple-import-sort/imports": "error",
```

Comment:
Just a heads-up.
I bumped `simple-import-sort`, which resulted in lint error something like "cannot find `simple-import-sort/sort` rules".
You will 100% get the same in the other projects when bumping it as well.